src/MsalRedirectAuthProvider.ts

```diff
@@ -24,46 +24,14 @@
 //
 
 import { IMsalAuthProviderConfig } from './Interfaces';
-import { Logger } from './logger';
 import { MsalAuthProvider } from './MsalAuthProvider';
 
-interface IRedirectLogin {
-  error: string;
-  errorDesc: string;
-  idToken: string;
-  tokenType: string;
-}
-
 export class MsalRedirectAuthProvider extends MsalAuthProvider {
-  private redirectLoginInfo: IRedirectLogin;
-
   constructor(authProviderConfig: IMsalAuthProviderConfig) {
     super(authProviderConfig);
-    if (this.redirectLoginInfo) {
-      if (this.redirectLoginInfo.idToken) {
-        this.acquireTokens(this.redirectLoginInfo.idToken);
-      } else if (this.redirectLoginInfo.errorDesc || this.redirectLoginInfo.error) {
-        Logger.error(
-          `Error doing login redirect; errorDescription=${this.redirectLoginInfo.errorDesc}, error=${
-            this.redirectLoginInfo.error
-          }`,
-        );
-      }
-    } else {
```

Comment:
I'm not sure what much of this is supposed to be doing. It never seemed to get executed during my tests, and I thought the base `MsalAuthProvider` class was meant to be responsible for fetching tokens. I removed this logic and brought the class closer in line with the Popup variation.